app/src/main/java/com/amaze/filemanager/ui/Extensions.kt

```diff
@@ -53,6 +55,23 @@ fun Context.startActivityCatchingSecurityException(intent: Intent) {
     }
 }
 
+/**
+ * Refactor the alias, based on if Amaze Utils is installed, then we disable the alias intent-filters
+ * if not installed then we enable the amaze utilities alias
+ */
+fun Context.refactorAUAlias(shouldEnable: Boolean) {
+    val component = ComponentName(this, "com.amaze.filemanager.amazeutilsalias")
+    if (!shouldEnable) {
```

Comment:
Both do the same thing?

Reply:
one enables the alias, other disables it

Reply:
Both branches have the same code.

Reply:
PackageManager.COMPONENT_ENABLED_STATE_DISABLED vs PackageManager.COMPONENT_ENABLED_STATE_ENABLED